<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_repo">
    <property name="title">
     <string>Хранилище</string>
    </property>
    <addaction name="action_repo_create"/>
    <addaction name="action_repo_open"/>
    <addaction name="action_repo_close"/>
    <addaction name="action_repo_add_item"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователь</string>
    </property>
    <addaction name="action_user_create"/>
    <addaction name="action_user_login"/>
    <addaction name="action_user_logout"/>
   </widget>
   <addaction name="menu_repo"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_repo_create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_repo_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_repo_close">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="action_repo_add_item">
   <property name="text">
    <string>Добавить элемент</string>
   </property>
  </action>
  <action name="action_user_create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_user_login">
   <property name="text">
    <string>Войти в хранилище</string>
   </property>
  </action>
  <action name="action_user_logout">
   <property name="text">
    <string>Выйти из хранилища</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
